### ISSUE TITLE
Fix Accessibility Labeling of SVG Logos

### ISSUE THREAD

**Issue by tomkeays** [tomkeays/lemoyne#55]
Problem and remediation described here: https://www.youtube.com/watch?v=ahYBjo0_rxw

The nutshell of the problem is: 
- The link requires an `aria-label` because there is no text inside the anchor element, only an SVG graphic. 
- The SVG, which itself has ARIA labeling, using the `aria-labelledby="falcone-logo-title falcone-logo-desc"`, needs to be removed from the accessibility tree using `aria-hidden="true"`. 

Here's the summary of the remediation that needs to be done:

```
<a href="https://resources.library.lemoyne.edu/library" aria-label="Noreen Reale Falcone Library">
  <svg id="falcone-svg-logo" viewBox="0 0 446 69" class="lmc-svg-banner-logo" 
       role="img" aria-labelledby="falcone-logo-title falcone-logo-desc" aria-hidden="true">
    <title id="falcone-logo-title">Noreen Reale Falcone Library</title>
    <desc id="falcone-logo-desc">Logotype of Noreen Reale Falcone Library</desc>
    <g>...</g>
  </svg>
</a>
```

Note also, the various paths inside the `<g>` element of the SVG, which define the individual letters of the logotype, lack `aria-label` attributes. Since the parent SVG is hidden, does this matter?  If not, what's the best way to label them:  `aria-label`  or `aria-labelledby`?  

Secondly, it turns out that the individual letters are not in proper sequence. I'll probably go ahead and reorder them, but does it actually matter?
